movierecommender.sql

```diff
@@ -0,0 +1,151 @@
+-- Create Database [movierecommender]
+CREATE database IF NOT EXISTS movierecommender
+DEFAULT CHARACTER SET utf8
+DEFAULT COLLATE utf8mb4_0900_ai_ci;
+
+SET default_storage_engine = InnoDB;
+
+
+-- Specify Database [movierecommender]
+USE movierecommender;
+
+
+-- Create Table [movies_metadata]
+CREATE TABLE `movierecommender`.`movies_metadata` (
```

Comment:
fix it by saying if exist then delete table, then create table, so if there is any error we need to modify the table, we will get error such like "table already exists"...same suggestion goes to all table below

Reply:
if your table already exists, drop it first using statement as below:
i.e DROP TABLE IF EXISTS `movierecommender`.`movies_metadata`;

Reply:
I have added DROP statement for each CREATE statement as comment, uncomment it as needed.